sql-statements/sql-statement-alter-table.md

```diff
@@ -157,7 +157,11 @@ Query OK, 0 rows affected, 1 warning (0.25 sec)
 
 The following major restrictions apply to `ALTER TABLE` in TiDB:
 
-- Changing the same column or index when altering multiple schema objects in a single statement is not supported.
+- When altering multiple schema objects in a single `ALTER TABLE` statement:
+
+    - Changing the same column or index in a single statement is not supported.
+    - TiDB validates statements according to the table schema **before execution**. For example, an error returns when executing `ALTER TABLE ADD INDEX i(b), DROP INDEX i;` due to the index `i` is not existed in the table.
```

Comment:
```suggestion
    - TiDB validates statements according to the table schema **before execution**. For example, an error returns when `ALTER TABLE ADD INDEX i(b), DROP INDEX i;` is executed because the index `i` does not exist in the table.
```